stdlib/public/Platform/libc-freebsd.modulemap.gyb

```diff
@@ -0,0 +1,311 @@
+//===--- libc-freebsd.modulemap.gyb ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Partial modulemap for libc on FreeBSD.
```

Comment:
Can you specify the version of FreeBSD this is derived from. I imagine that over time this may change so it would be useful to start the culture of documenting this. Maybe just putting that in the commit msg would be sufficient. Your thoughts?